oeps/processes/oep-0053-proc-package-release.rst

```diff
@@ -0,0 +1,136 @@
+| Type          | Process                                                      |
++---------------+--------------------------------------------------------------+
+| Created       | 2021-04-07                                                   |
++---------------+--------------------------------------------------------------+
+| Review Period |                                                              |
++---------------+--------------------------------------------------------------+
+| References    |                                                              |
++---------------+--------------------------------------------------------------+
+
+Abstract
+========
+
+This document proposes a standard package release process to be followed across the Open edX community.
+
+Motivation
+==========
+
+We recently shifted the CI from Travis to Github Actions which led us to move the package release process to Github Actions as well. During this process,
+we observed some problems in the current package release process. The package release process is comprised of several manual steps.
+Once the changes are pushed, we've to update the changelogs, bump the version number then create a tag or Github release which finally triggers the final package release.
+As all these steps need to be done manually, sometimes things get missed. For instance, you made changes, bumped the version, and created a release but forgot
+to add changelogs. Moreover, we prefer to publish releases on tags but also want to keep Github releases aligned with PyPI releases so that release history
+could be found in the GitHub repository as well.
+
+To solve these above-mentioned issues, we have automated the package release process. Initially, we looked for packages that could do the job but the
+available packages don't provide much control so we created a pipeline using a set of Github workflows to achieve the goal. By using the release process
+proposed in the OEP we’ll be able to achieve the following:
+
+* Just push the changes and the release pipeline will automatically take care of the rest of the process.
+* Trigger release on tag creation
+* Keep Github releases aligned with the PyPI releases
+* Encourage usage of conventional commits
+
+Specification
+=============
+
+Github has a feature to create workflow templates to be used across the organization, which helps standardize the processes. We’ve written some workflow
+templates to automate the package release process in edX owned python packages. Some of these templates use reusable Github workflows and custom Github actions
+to achieve the goal while keeping a single source of truth and making workflow templates simple.
+
+How to set up?
+--------------
+
+To set up the release process in python packages, you need to add the following workflow templates in package’s repository
+(if you need help for using workflow templates, see the `documentation`_):
```

Comment:
You should specify where these workflows will live.  I think `openedx/.github` would be the right repo.  The templates can be in both `edx/.github` and `openedx/.github`.

Reply:
I've attached the urls for these workflows in headings below